src/components/common/popup.jsx

```diff
@@ -1,69 +1,119 @@
 import * as React from 'react'
 import PropTypes from 'prop-types'
-import Modal from 'react-modal'
```

Comment:
why not to use it?

Reply:
Fixed. I thought that I am able to create own modal, but as always forgot about a11y 😄